doc/changes.rst

```diff
@@ -5,7 +5,10 @@ Change Log
 3.2.5 (unreleased)
 ------------------
 
-* No changes yet.
+* Update :meth:`desiutil.depend.add_dependencies` to include key environment
```

Comment:
It is safer to use `:func:` rather than `:meth:` because the latter is meant to refer to methods of objects defined in a `class`.

Reply:
done; thanks for the explanation